### ISSUE TITLE
Emoji not display in list

### ISSUE THREAD

**Issue by ArmeF97** [UnigramDev/Unigram#938]
![immagine](https://user-images.githubusercontent.com/6240983/50480964-53cfe000-09df-11e9-8c03-fe5f54e3a278.png)

I'm using Windows 10 version 1803 build 17134.472

**Comment by MarcAnt01** [comment]
What do you mean by display list?

**Comment by ArmeF97** [comment]
Maybe now the title explains it better... see the white squares?

**Comment by MarcAnt01** [comment]
No problem on 1809

**Comment by ArmeF97** [comment]
Yes but there is a problem on 1803, it's a bug

**Comment by ArmeF97** [comment]
So Unigram is for latest Win 10 only?

**Comment by FrayxRulez** [comment]
It is optimized for the latest version of the OS as any app should be :)

**Comment by ArmeF97** [comment]
Ok, thanks

**Comment by callummoffat** [comment]
No, but when using the emoji list only the emoji supported by the version of Windows you are on will appear.


Callum Moffat

________________________________
